test-case/run-all-tests.sh

```diff
@@ -94,99 +128,99 @@ test_sof-logger()
 }
 test_ipc-flood()
 {
-	"$mydir"/check-ipc-flood.sh -l 10
+	"$mydir"/check-ipc-flood.sh -l "$small_loop"
 }
 test_playback-d100l1r1()
 {
-	"$mydir"/check-playback.sh -d 100 -l 1 -r 1
+	"$mydir"/check-playback.sh -d ${long_duration} -l 1 -r 1
 }
 test_capture-d100l1r1()
 {
-	"$mydir"/check-capture.sh -d 100 -l 1 -r 1
+	"$mydir"/check-capture.sh -d ${long_duration} -l 1 -r 1
 }
 test_playback-d1l100r1()
 {
-	"$mydir"/check-playback.sh -d 1 -l 100 -r 1
+	"$mydir"/check-playback.sh -d 1 -l "$large_loop" -r 1
 }
 test_capture_d1l100r1()
 {
-	"$mydir"/check-capture.sh -d 1 -l 100 -r 1
+	"$mydir"/check-capture.sh -d 1 -l "$large_loop" -r 1
 }
 test_playback_d1l1r50()
 {
-	"$mydir"/check-playback.sh -d 1 -l 1 -r 50
+	"$mydir"/check-playback.sh -d 1 -l 1 -r "$large_round"
 }
 test_capture_d1l1r50()
 {
-	"$mydir"/check-capture.sh -d 1 -l 1 -r 50
+	"$mydir"/check-capture.sh -d 1 -l 1 -r "$large_round"
 }
 test_speaker()
 {
-	"$mydir"/test-speaker.sh -l 50
+	"$mydir"/test-speaker.sh -l "$medium_loop"
 }
 test_pause-resume-playback()
 {
-	"$mydir"/check-pause-resume.sh -c 100 -m playback
+	"$mydir"/check-pause-resume.sh -c "$large_count" -m playback
 }
 test_pause-resume-capture()
 {
-	"$mydir"/check-pause-resume.sh -c 100 -m capture
+	"$mydir"/check-pause-resume.sh -c "$large_count" -m capture
 }
 test_volume()
 {
-	"$mydir"/volume-basic-test.sh -l 100
+	"$mydir"/volume-basic-test.sh -l "$large_loop"
 }
 test_signal-stop-start-playback()
 {
-	"$mydir"/check-signal-stop-start.sh -m playback -c 50
+	"$mydir"/check-signal-stop-start.sh -m playback -c "$medium_count"
 }
 test_signal-stop-start-capture()
 {
-	"$mydir"/check-signal-stop-start.sh -m capture -c 50
+	"$mydir"/check-signal-stop-start.sh -m capture -c "$medium_count"
 }
 test_xrun-injection-playback()
 {
-	"$mydir"/check-xrun-injection.sh -m playback -c 50
+	"$mydir"/check-xrun-injection.sh -m playback -c "$medium_count"
 }
 test_xrun-injection-capture()
 {
-	"$mydir"/check-xrun-injection.sh -m capture -c 50
+	"$mydir"/check-xrun-injection.sh -m capture -c "$medium_count"
 }
 test_simultaneous-playback-capture()
 {
-	"$mydir"/simultaneous-playback-capture.sh -l 50
+	"$mydir"/simultaneous-playback-capture.sh -l "$medium_loop"
 }
 test_multiple-pipeline-playback()
 {
-	"$mydir"/multiple-pipeline-playback.sh -l 50
+	"$mydir"/multiple-pipeline-playback.sh -l "$medium_loop"
 }
 test_multiple-pipeline-capture()
 {
-	"$mydir"/multiple-pipeline-capture.sh -l 50
+	"$mydir"/multiple-pipeline-capture.sh -l "$medium_loop"
 }
 test_multiple-pause-resume()
 {
 	"$mydir"/multiple-pause-resume.sh -r 25
 }
 test_kmod-load-unload()
 {
-	"$mydir"/check-kmod-load-unload.sh -l 50
+	"$mydir"/check-kmod-load-unload.sh -l "$medium_loop"
 }
 test_kmod-load-unload-after-playback()
 {
-	"$mydir"/check-kmod-load-unload-after-playback.sh -l 15
+	"$mydir"/check-kmod-load-unload-after-playback.sh -l "$small_loop"
```

Comment:
also we could disable these kmod tests with an option since they don't work on all devices (e..g Cyan)

Reply:
How are they skipped in CI? If they SKIP with  `exit 2` then maybe we could catch this here just like Jenkins does.... in a later PR.

We also need to be conscious not to turn this hack into yet another NIH test runner.